packages/ember-core/src/components/sidebar.gts

```diff
@@ -135,6 +166,19 @@ export class Group extends Component<GroupSignature> {
     );
   }
 
+  get models() {
```

Comment:
Could you add the same/similar assertion that Ember has, making sure that both `model` and `models` aren't provided?